operations/onboarding.md

```diff
@@ -0,0 +1,63 @@
+   Their job is to carry out the process described in the onboarding issue template.
+2. **Open an onboarding issue in the Team Compass**.
+   This issue will track the onboarding process, and serves as the {term}`Source of Truth` for steps to take in order to onboard a new team member.
+
+   ```{button-link} https://github.com/2i2c-org/team-compass/issues/new?assignees=&labels=type%3A+onboard&template=new-team-member.md&title=Onboarding+%3Cname%3E
+   :color: primary
+   Open an onboarding issue
+   ```
+
+## Our documentation structure
+
+Documentation is very important to 2i2c, because it is the way that we share information across our asynchronous and distributed team.
+Anything worth sharing once is worth documenting for all.
+
+Our documentation all uses the same set of tools, which all team members need to be familiar with.
+
+See [our documentation guide](../reference/documentation/overview.md) for an overview of our documentation workflow and our major sources of documentation.
+
+
+## How our team compass is organized
+
+[Our team compass](https://compass.2i2c.org) is the source of truth for all of our practices.
+It is roughly divided into three sections:
+
+- **Organization-wide sections** cover information about 2i2c itself. They are generally managed by our {role}`Executive Director`.
+- **Functional areas** focus on particular aspects of our operation, and are usually managed by a functional lead.
+- **Managed JupyterHub Service** is a special section that has a lot of cross-area documentation focused around our hub service.
+  It's in a dedicated section because it doesn't really belong in any one functional area.
+- **Reference documentation** is a place to collect many pieces of information in one place, and for extra documentation that is useful across the organization.
+
+Sections of our Team Compass should roughly follow the same high-level structure.
+For example, most of our functional areas have a "Structure" section that describes their roles and team structure, as well as a "Workflow" section that describes their workings in more detail.
+
+
+## Where to learn about our workflow
+
+As a distributed team, we have a few practices that are geared towards keeping everybody on the same page given that we're in many different time zones.
+Here are the right places to learn more about our workflow:
+
+- [Our organization-wide section](../organization/index.md) has an overview of our mission, structure, and strategy.
+- [Our team operations section](../operations/index.md) has an overview of our organization-wide practices and workflow.
+- [Our types of meetings page](../operations/meetings.md) describes the kinds of meetings we have across 2i2c.
+  Different teams each interpret these in a different way, but this covers the basics.
+
+Beyond these, your workflow will likely depend on the functional area where you work.
```

Comment:
This section seems to address a new member. I wonder if it might be worth splitting the doc into clear sections one for the onboarding champion and those that orient the new team member.